### ISSUE TITLE
Icons in sidebar fall outside the range on macOS

### ISSUE THREAD

**Issue by CSY54** [getferdi/ferdi#1894]
**Bug description:**

The icons in sidebar fall outside the range of sidebar. (see screenshot)

It seems after setting `padding-top: 23px;` of `.darwin .sidebar` in [src/styles/layout.scss](https://github.com/getferdi/ferdi/blob/develop/src/styles/layout.scss#L118), the CSS of `height`, `padding-top`, `margin-top` generated by [generateVerticalStyle](https://github.com/getferdi/ferdi/blob/develop/src/features/appearance/index.js#L131) should add 23px as well.

Also, during finding the problem, I noticed there is a typo in the CSS of [.workspaces-drawer](https://github.com/getferdi/ferdi/blob/develop/src/features/appearance/index.js#L161).

**Steps to reproduce:**

Toggle "Use horizontal style" on.

**Expected behavior**

The icons should be shown within sidebar.

**Screenshots**

<img width="200" src="https://user-images.githubusercontent.com/18496305/132573947-a6599538-a131-473f-80e9-d977256388c5.png">

After modifying CSS (the part with +/-23px is where I modified):
<img width="325" src="https://user-images.githubusercontent.com/18496305/132579368-a64eafdf-e755-45af-9f44-3bcdea3dfd86.png">
<img width="200" src="https://user-images.githubusercontent.com/18496305/132578245-9d1fa369-c26a-489d-81d7-4007b17abc6e.png">

**Environment:**

- Operating System: macOS
- Ferdi Version: 5.6.1-beta.3
- Server: Using without an account

**Comment by vraravam** [comment]
Thanks for the fix @CSY54 - would you want to take the opportunity to create a PR for the same? Or shall I do it?

**Comment by CSY54** [comment]
Maybe I can try it

**Comment by vraravam** [comment]
[patch.txt](https://github.com/getferdi/ferdi/files/7133121/patch.txt)

You can use this patch file to raise the PR.

**Comment by CSY54** [comment]
Wow, you're quick, I've just successfully started the dev server.

I think there might be a problem with your patch. The padding of 23px was only applied on macOS, so the CSS should be specified to `.darwin`.

i.e. Instead of setting
```css
.app .app__content {
  padding-top: ${sidebarWidth + verticalStyleOffset}px !important;
}
```
maybe set it like this
```css
.app .app__content {
  padding-top: ${sidebarWidthStr} !important;
}

.darwin .app .app__content {
  padding-top: ${sidebarWidth + verticalStyleOffset}px !important;
}
```

I have no other OS to test for it right now, so maybe limit it to macOS would be better for now?

**Comment by vraravam** [comment]
that's ok - we can assume that the issue was present in other OSes as well for now. Please go ahead and raise the PR

**Comment by CSY54** [comment]
There's one more issue: the height of the sidebar is currently set to 64px, but the height of the items in the sidebar (`.tab-item`) are set to 65px, making it overflow and scrollable. So, I'm setting the height of the sidebar into `` `${sidebarWidth + verticalStyleOffset + 1}px !important;` ``, is this ok?

**Comment by vraravam** [comment]
Sure - please go ahead with that change

**Comment by vraravam** [comment]
Fixed by #1896  - closing.
